compiler/aarch64/codegen/FPTreeEvaluator.cpp

```diff
@@ -82,21 +82,136 @@ OMR::ARM64::TreeEvaluator::dbits2lEvaluator(TR::Node *node, TR::CodeGenerator *c
    return trgReg;
    }
 
+static uint16_t
+getImm8forFloat(float value)
+   {
+   if(value == 0.0f)
+      {
+      return 256;
```

Comment:
You should not use 256 as the error here.
256 (0xFF) is used for encoding -1.9375 in the AArch64 8-bit modified immediate constants.

Reply:
My comment above was wrong: 256 is not 0xFF.